README.md

```diff
@@ -138,6 +138,11 @@ code examples in our YARD documentation. To run `doctest` locally:
 bundle exec yard doctest
 ```
 
+## Logo
+
+`MemoWise`'s logo was created by Luci Cooke. The logo is licensed under a
```

Comment:
Up to Luci of course but she may want her name to be a link, like:

```
created by [Luci Cooke](https://github.com/lmcooke)
```

(or a link to a different website of her choosing).